tomcat9-maven-plugin/src/main/java/org/apache/tomcat/maven/plugin/tomcat9/run/RunMojo.java

```diff
@@ -479,18 +480,23 @@ private WebResource urlToWebResource( URL url, String path )
                                 jarFile = new JarFile( filePath );
 
                                 JarEntry jarEntry = jarFile.getJarEntry( StringUtils.removeStart( path, "/" ) );
-
+/*
                                 return new JarResource( this, //
                                                         getPath(), //
                                                         filePath, //
                                                         url.getPath().substring( 0, idx ), //
                                                         jarEntry, //
                                                         "", //
                                                         null );
+                                */
+                                
+                                JarResourceSet jr = new JarResourceSet(context.getResources(), getPath(), filePath, getPath());
```

Comment:
original internal path is `""`, and set to `getPath()`, please verify as `"/"`, or `getPath()`.

Reply:
@liudongmiao I think that particular line might have gone in the merge from my code... if not, could you please explain what you're seeing that's wrong here? The `JarResourceSet` parameters are a little opaque to me, so I'd appreciate some more explanation!

Reply:
Sorry for outdated.

Yes, for the code, it's changed from `getPath()` to `"/"` in commit 7f26908685ee565c1a501ec7b03763905adf1b1b.
And the internal path is always `"/"` now.

Reply:
I'm going to upgrade from tomact7 to tomcat9.
now I'm using tomcat7 mvn plugin for run & debug in local machine. 
I want to use your tomcat9 mvn plugin in order to run & debug. 
How can I use it? 
Thanks a lot